pkg/virt-controller/watch/preset_test.go

```diff
@@ -1034,21 +993,23 @@ var _ = Describe("VirtualMachineInstance Initializer", func() {
 				},
 			}
 
-			// Register the expected REST call
-			server.AppendHandlers(
-				ghttp.CombineHandlers(
-					ghttp.VerifyRequest("PUT", "/apis/kubevirt.io/v1alpha2/namespaces/default/virtualmachineinstances/testvmi"),
-					ghttp.RespondWithJSONEncoded(http.StatusOK, vmi),
-				),
-			)
+			virtClient.EXPECT().VirtualMachineInstance(k8smetav1.NamespaceDefault).Return(vmiInterface)
 
-			err := app.vmiPresetController.initializeVirtualMachine(vmi)
+			vmiInterface.EXPECT().Update(gomock.Any()).Do(func(arg interface{}) {
+				vmi := arg.(*v1.VirtualMachineInstance)
+				disk := vmi.Spec.Domain.Devices.Disks[0]
+				Expect(disk.Disk).ToNot(BeNil(), "DiskTarget should not be nil")
+				Expect(disk.Disk.Bus).ToNot(BeEmpty(), "DiskTarget's bus should not be empty")
 
-			Expect(err).ToNot(HaveOccurred())
+				// The copy of the VMI sent to the server should be initialized
+				_, found := vmi.Annotations[initializerMarking]
+				Expect(found).To(BeTrue(), "vmi should have been initialized")
 
-			disk := vmi.Spec.Domain.Devices.Disks[0]
-			Expect(disk.Disk).ToNot(BeNil(), "DiskTarget should not be nil")
-			Expect(disk.Disk.Bus).ToNot(BeEmpty(), "DiskTarget's bus should not be empty")
+				Expect(vmi.Status.Phase).ToNot(Equal(v1.Failed))
+			}).Return(nil, nil)
+
+			err := vmiPresetController.initializeVirtualMachine(vmi)
```

Comment:
app.vmiPresetController ?

Reply:
What do you mean? anything referencing `app` was removed.

Reply:
Yeah sorry, my bad.  I missed the removal at L#1013, but figured it out when going through the full file.  Meant to cancel that comment but was exiting everything out in a hurry and added instead :)  My bad.